### DISCUSSION TITLE
how to enable grub-btrfs with dinit

### DISCUSSION THREAD

**Discussion by Vrashabh-Sontakke** [Antynea/grub-btrfs#316] [Q&A]
I am using Artix Linux with "dinit" as the init system (no-systemd).
How to enable grub-btrfsd or grub-btrfs.path service ?
I tried these commands with no luck :
dinitctl enable grub-btrfsd
dinitctl enable grub-btrfs.path

**Comment by Schievel1** [comment]
I think you need to write you own service file that starts `grub-btrfsd`.
See here: https://davmac.org/projects/dinit/man-pages-html/dinit-service.5.html

I think the `type` should be `process` here and the `env-file` maybe `/etc/default/grub-btrfs/config`. And `command` something like `/usr/bin/grub-btrfsd --syslog /.snapshots`. Or something like that, depending on your configuration. 

We don't have any service files for dinit, because we don't use it and therefore can't test them. In fact I haven't even heard of this before. If you have a working service file for dinit, a PR would be appreciated.
